scout/parse/models.py

```diff
@@ -281,15 +281,15 @@ class ScoutLoadConfig(BaseModel):
     gene_fusion_report: Optional[str] = None
     gene_fusion_report_research: Optional[str] = None
     gene_panels: Optional[List[str]] = []
-    genome_build: int = Field([], alias="human_genome_build")
+    genome_build: int = Field(38, alias="human_genome_build")
     individuals: List[ScoutIndividual] = Field([], alias="samples")
     lims_id: Optional[str] = None
-    madeline_info: Optional[str] = Field("", alias="madeline")  #!
+    madeline_info: Optional[str] = Field(None, alias="madeline")  #!
     multiqc: Optional[str] = None
     owner: str = None
     peddy_ped: Optional[str] = None
-    peddy_ped_check: Optional[str] = Field("", alias="peddy_check")
-    peddy_sex_check: Optional[str] = Field("", alias="peddy_sex")
+    peddy_ped_check: Optional[str] = Field(None, alias="peddy_check")
+    peddy_sex_check: Optional[str] = Field(None, alias="peddy_sex")
     phenotype_terms: Optional[List[str]] = None
     rank_model_version: Optional[str] = ""
```

Comment:
What about this and the sv_rank_model_version? None seems more prudent, but I haven't checked in the templates/controllers.

Reply:
I was thinking the same, I'll check!

Reply:
After setting them to None, and trying to load demo case, the values are still correctly parsed:

![image](https://user-images.githubusercontent.com/28093618/152961920-f74a2b35-25ae-43aa-bab0-2d1169eb1bbd.png)

Reply:
If case config.yaml doesn't contain any rank_model_version or sv_rank_model_version, then the created document case doesn't contain these keys, which I think is fine.